src/vs/platform/backup/electron-main/backupMainService.ts

```diff
@@ -21,13 +23,29 @@ export class BackupMainService implements IBackupMainService {
 	private workspacesJsonContent: IBackupWorkspacesFormat;
 
 	constructor(
-		@IEnvironmentService environmentService: IEnvironmentService
+		@IEnvironmentService environmentService: IEnvironmentService,
+		@ILifecycleService lifecycleService: ILifecycleService
 	) {
 		this.backupHome = environmentService.backupHome;
 		this.workspacesJsonPath = environmentService.backupWorkspacesPath;
+
+		if (lifecycleService) {
```

Comment:
@bpasero is this fine to go into `serviceTestUtils.ts`? It would be the first `electron-main` file there. From what I can see there is very little to no testing of code in `vs/code` of which this service has a dependency.